frontend/src/pages/user-profile/components/user-profile-sidebar/user-profile-sidebar.tsx

```diff
@@ -75,15 +76,17 @@ const UserProfileSidebar: React.FC<Properties> = ({
         <div className={styles['buttons-container']}>
           <div className="visually-hidden">Profile settings options</div>
           {SETTINGS_OPTIONS.map((option) => {
+            // TODO: fix isActive to show proper result
```

Comment:
That is temporary for current PR?

Reply:
actually, I thought that merging https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/343 would help me fix this, cause @OlesiaTr is working on helper to get selected route. I can try to fix this without the issue mentioned above

Reply:
I fixed this. I am not sure that this is the best option, but it is what I came up with) Maybe you have a better suggestion?